querybook/server/datasources/query_execution.py

```diff
@@ -370,18 +370,44 @@ def export_statement_execution_result(
         api_assert(valid, "Invalid exporter params, reason: " + reason)
 
     task = export_query_execution_task.apply_async(
-        args=[
-            exporter.exporter_name,
-            statement_execution_id,
-            current_user.id,
-            originator,
-        ],
+        args=[exporter.exporter_name, statement_execution_id, current_user.id,],
```

Comment:
instead of having exporter_params passed as kwargs, can we have a parameter called exporter_params instead?